core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -16,6 +16,8 @@
  * @fileoverview Utility service for the learner's view of an exploration.
  */
 
+oppia.constant('MIN_TIME_FOR_ACTUAL_START_SECS', 60);
```

Comment:
Add one-line comment to explain the meaning of this constant (and the concept of "actual start").

Reply:
Redundant since we plan to record actual start on just the initial state transition.